src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -305,16 +282,12 @@ public async Task DownloadQualityProfile_WhenQualityProfileIsNotAvailable_Fails(
             var notifications = new ConfigurableProgressStepExecutionEvents();
             var progressAdapter = new FixedStepsProgressAdapter(notifications);
 
+            // CSharp is a supported language, but only the VB.NET QP is available
```

Comment:
Misc: new comment explaining the setup of the test. Not related to the current changes.